test/tests_for_sdkmr/ee2_scheduler_test.py

```diff
@@ -16,63 +29,51 @@
-        rr = CatalogUtils.normalize_job_settings(cgroups)
-
-        print(rr)
-        params["cg_resources_requirements"] = rr
-
-        return params
 
-    def test_empty_params(self):
-        c = self.condor
-        params = {"job_id": "test_job_id", "user_id": "test", "token": "test_token"}
-        with self.assertRaisesRegex(
-            Exception, "cg_resources_requirements not found in params"
-        ):
-            c._create_submit(params)
+    def _create_sample_params(self, request_x, scheduler_requirements=None):
+        sr = scheduler_requirements if scheduler_requirements else {}
+        return JobSubmissionParameters(
+            self.job_id,
+            AppInfo("fake.fake", "fake/app"),
+            JobRequirements(
+                request_x.get("request_cpus", 101),
+                request_x.get("request_memory", 102),
+                request_x.get("request_disk", 103),
+                request_x.get("client_group", "defaultcg"),
+                request_x.get("client_group_regex", True),
+                bill_to_user=request_x.get("bill_to_user"),
+                ignore_concurrency_limits=request_x.get(
+                    "ignore_concurrency_limits", False
+                ),
+                scheduler_requirements=sr,
+            ),
+            UserCreds(self.user, "test_token"),
+        )
 
     def test_create_submit_file(self):
         # Test with empty clientgroup
         logging.info("Testing with njs clientgroup")
         c = self.condor
-        params = self._create_sample_params(cgroups=["njs"])
+        params = self._create_sample_params({"client_group": "njs"})
 
         default_sub = c._create_submit(params)
 
         sub = default_sub
         self.assertEqual(sub["executable"], c.initial_dir + "/" + c.executable)
-        self.assertEqual(sub["arguments"], f"{params['job_id']} {c.ee_endpoint}")
+        self.assertEqual(sub["arguments"], f"{self.job_id} {c.ee_endpoint}")
         self.assertEqual(sub["universe"], "vanilla")
-        self.assertEqual(sub["+AccountingGroup"], '"' + params["user_id"] + '"')
-        self.assertEqual(sub["Concurrency_Limits"], params["user_id"])
+        self.assertEqual(sub["+AccountingGroup"], f'"{self.user}"')
+        self.assertEqual(sub["Concurrency_Limits"], self.user)
         self.assertEqual(sub["+Owner"], '"condor_pool"')
         self.assertEqual(sub["ShouldTransferFiles"], "YES")
         self.assertEqual(sub["When_To_Transfer_Output"], "ON_EXIT_OR_EVICT")
 
-        self.assertEqual(sub[Condor.REQUEST_CPUS], c.config["njs"][Condor.REQUEST_CPUS])
-        self.assertEqual(
-            sub[Condor.REQUEST_MEMORY], c.config["njs"][Condor.REQUEST_MEMORY]
-        )
-        self.assertEqual(sub[Condor.REQUEST_DISK], c.config["njs"][Condor.REQUEST_DISK])
+        self.assertEqual(sub[REQUEST_CPUS], "101")
+        self.assertEqual(sub[REQUEST_MEMORY], "102MB")
+        self.assertEqual(sub[REQUEST_DISK], "103GB")
```

Comment:
This was a bit tricky to figure out as there is conflicting info as to what HTC expects here:
- the concierge params are simply `int`s and are not suffixed with units AFAICT.
- clientgroup params from the catalog or the `deploy.cfg` file are always treated as strings, and the last character of `memory` and last two characters of `disk` are trimmed off before saving to mongo (but not when submitted to HTC).
- the catalog doesn't use units when specifying memory or disk
- the `deploy.cfg` always uses units when specifying memory or disk

As such, it's hard to know what is the correct input. Since the deploy.cfg values do have unit appended, I judged that units must work, and therefore including units consistently should be safe.